es5.md

```diff
@@ -531,7 +531,7 @@ A unary operator token.
 interface UpdateExpression <: Expression {
     type: "UpdateExpression";
     operator: UpdateOperator;
-    argument: Expression;
+    argument: Pattern | CallExpression;
```

Comment:
Not just CallExpression. Any expression can be in this position in ES5 and below.